<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdClassClass</class>
 <widget class="QWidget" name="ThresholdClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThresholdClass</string>
  </property>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>阈值变换参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阈值：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valuelineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>算法</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;说明：&lt;/span&gt;三个算法当中，算出最佳阈值会直接在上面显示处理好的图像。&lt;/p&gt;&lt;p&gt;阈值也可以用户自定义设置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_showProcessImage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>461</width>
     <height>461</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>351</y>
     <width>123</width>
     <height>92</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="loopradioButton">
      <property name="text">
       <string>迭代法</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="OTSUradioButton">
      <property name="text">
       <string>OTSU法</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="selfradioButton">
      <property name="text">
       <string>自适应阈值分割</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="triangleradioButton">
      <property name="text">
       <string>最大熵法(自定义)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="thresvaluehorizontalSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
